src/theia/diagram-widget.ts

```diff
@@ -162,6 +162,8 @@ export class DiagramWidget extends BaseWidget implements StatefulWidget, Navigat
         const svgElement = this.node.querySelector(`#${this.viewerOptions.baseDiv} svg`) as HTMLElement;
         if (svgElement !== null)
             svgElement.focus();
+		else
+			this.node.focus();
```

Comment:
`div` nodes do not accept focus by default. We could work around that by setting `tabindex` to `"-1"` before calling `focus()`.
See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
Fixed.